llvm/lib/Target/AMDGPU/VOPCInstructions.td

```diff
@@ -87,6 +87,17 @@ class VOPC_Profile<list<SchedReadWrite> sched, ValueType vt0, ValueType vt1 = vt
 multiclass VOPC_Profile_t16<list<SchedReadWrite> sched, ValueType vt0, ValueType vt1 = vt0> {
   def NAME : VOPC_Profile<sched, vt0, vt1>;
   def _t16 : VOPC_Profile<sched, vt0, vt1> {
+    let IsTrue16 = 1;
+    let IsRealTrue16 = 1;
+    let Src1RC32 = getVregSrcForVT<Src1VT, 1/*IsTrue16*/, 1/*IsFake16*/>.ret;
```

Comment:
I guess the `1/*IsFake*/` bits is what the description refers to explaining that the profiles are going to be updated later to use actual t16 operands?

Reply:
Yes. The _t16 will be updated in the following patches to use true16 profile